gapis/api/gles/custom_replay.go

```diff
@@ -39,39 +39,39 @@ type objectKey struct {
 func (i BufferId) remap(cmd api.Cmd, s *api.GlobalState) (key interface{}, remap bool) {
```

Comment:
> The reference tracking on maps should be on the actual map.

Should it? The only other field in the map type was the arena, and identical maps should be from the same arena.
Not sure what this is fixing. FWIW, the map will become a cgo type and so private, where this change will have to be reverted.
More context on what you are attempting to fix here would be helpful.